packages/core/src/io/reader-context.ts

```diff
@@ -20,6 +20,7 @@ export class ReaderContext {
 	public skins: Skin[] = [];
 	public animations: Animation[] = [];
 	public scenes: Scene[] = [];
+	public scene?: number;
```

Comment:
It doesn't seem like `context.scene` is actually used, maybe this could be omitted until it's needed? If your project needs it for some reason that's OK too.